README.md

```diff
@@ -173,6 +173,30 @@ Using the `Parameters` classes is not required, but they can help in a number of
 - Provides a more natural way to construct parameters
 - Facilitates ASP.NET Core model binding (bind an HTML form to a `Parameters` instance)
 
+## Debugging
+
+### View HTTP Requests
+
+Users can audit the HTTP requests being made by the library by setting the `RequestAuditor` property of a `ClientConfiguration` with an `Action` that accepts an `HttpRequestMessage`. 
+
+The `Action` will be executed with the pre-flight `HttpRequestMessage` before the request is executed. At that point, the `HttpRequestMessage` is configured with all expected data (headers, body, etc.).
+
+Users can interact with the `HttpRequestMessage` inside the `Action` as they see fit (e.g. logging).
+
+```csharp
+void myRequestAuditingAction(HttpRequestMessage request)
+{
+    // Interact with the HttpRequestMessage here
+}
+
+Client client = new Client(new ClientConfiguration("EASYPOST_API_KEY")
+{
+    RequestAuditor = myRequestAuditingAction,
```

Comment:
thought: I'd love to see a dead simple real example. If I'm looking at this, I wouldn't know what I could do. Can we make it as simple as printing to console in the example so people know exactly how to access the request?